app/models/item.rb

```diff
@@ -8,4 +8,10 @@ class Item < ActiveRecord::Base
   def price
     sprintf('%.2f', read_attribute(:price))
   end
+
+  def calculate_average_review_score
+    rating_sum = reviews.inject(0) { |sum, review| sum + review.rating }
+    rating_count = reviews.count
+    (rating_sum.to_f / rating_count).round(2)
+  end
```

Comment:
`rating_count = reviews.count` is pretty redundant - it doesn't really shorten up our code at all.. Compare above to:

```
+  def calculate_average_review_score
 +    rating_sum = reviews.inject(0) { |sum, review| sum + review.rating }
 +    (rating_sum.to_f / reviews.count).round(2)
 +  end
```